.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
**PLEASE NOTE: I'd use this inline comment feature to add all the changes you should implement**

**[REQUIRED CHANGES]**

**YOUR PR**

- Please, kindly add a summary describing introduced changes in more detail to make your PR more descriptive.
- You could update your PR title to something like `Hello Microverse` as this is more descriptive in reference to your project.
- Description: could be something like this
```
In this project milestone, I did the following:
- Added linter files.
- Added `test.md` file, and `node_modules/` directory in the `.gitignore` file
- Added an `h1` in the `index.html` file
```

**README FILE**

- Please, kindly update your readme by adding the missing header from the [microverse readme template](https://github.com/microverseinc/readme-template)

**.gitignore file**

- Please, kindly add the `node_modules/` directory also.

**LINTER FILE**
- Please, kindly add the missing linter files to your project from the [microverse html-css linter file](https://github.com/microverseinc/linters-config/tree/master/html-css). Also, ensure there are no linters error in your project.

Reply:
Hello @zieeco 
I made the changes as requested. Kindly go through them and advise.

Thank You!